sal-service/src/main/java/org/ow2/proactive/sal/service/rest/PersistenceRest.java

```diff
@@ -22,15 +24,53 @@
 
 @RestController
 @RequestMapping(value = "/persistence")
-@Api(description = "Operations on the DB", consumes = MediaType.APPLICATION_JSON, produces = MediaType.APPLICATION_JSON)
+@Api(description = "Operations to clean the cloud, clusters and SAL database", consumes = MediaType.APPLICATION_JSON, produces = MediaType.APPLICATION_JSON)
 public class PersistenceRest {
 
     @Autowired
     private RepositoryService repositoryService;
```

Comment:
You should remove this autowired service and call` repositoryService.cleanAll(sessionId);` from PersistenceService persistenceService